docs/src/code-style.rst

```diff
@@ -58,10 +58,13 @@ Run pylint with our developer tool :github-source:`tools/pylint.sh`::
 
     ./tools/pylint.sh
 
-When you think a warning is a false positive, add a comment before the specific line::
+When you think a warning is a false positive, add a comment::
```

Comment:
What about linking straight to the pylint documentation where different forms and their effects are explained? The section [has a convenient label](https://github.com/pylint-dev/pylint/blob/main/doc/user_guide/messages/message_control.rst?plain=1#L26), so we should be able to just reference it like this:

```suggestion
When you think a warning is a false positive, add a comment (see :doc:`pylint:block_disables`)::
```

…if we add their docs to our [intersphinx mapping](https://github.com/digitalfabrik/integreat-cms/blob/develop/docs/src/conf.py#L75) in `docs/src/conf.py`:

```diff
     "dateutil": ("https://dateutil.readthedocs.io/en/stable/", None),
     "geopy": ("https://geopy.readthedocs.io/en/stable/", None),
     "lxml": ("https://lxml.de/apidoc/", None),
+    "pylint": ("https://pylint.readthedocs.io/en/latest/", None),
     "python": (
         f"https://docs.python.org/{sys.version_info.major}.{sys.version_info.minor}/",
         None,
```

Reply:
@MizukiTemma Sorry, I got confused about sphinx's roles in cross referencing. It should be `` :doc:`pylint:block_disables` `` (I checked this time)

I will not be able do a full review of the PR though, sorry